src/schemas/f142/f142_rw.cpp

```diff
@@ -142,11 +144,13 @@ void HDFWriterModule::parse_config(std::string const &ConfigurationStream,
           find<std::string>("source", ConfigurationStreamJson)) {
     SourceName = SourceNameMaybe.inner();
   } else {
-    LOG(Sev::Error, "Key \"source\" is not specified in json command");
+    Logger->error("Key \"source\" is not specified in json command");
     return;
   }
 
-  if (!findType(ConfigurationStreamJson, TypeName)) {
+  if (auto TypeNameMaybe = find<std::string>("type", ConfigurationStreamJson)) {
```

Comment:
This looks like a mistake again, the `dtype` support change is being reverted in this branch for some reason.